### DISCUSSION TITLE
time crate for date feature

### DISCUSSION THREAD

**Discussion by Cyrix126** [mikaelmello/inquire#221] [Show and tell]
Hello, I made a fork for the feature date using the [time](https://crates.io/crates/time) crate  instead of [chrono](https://crates.io/crates/chrono).

https://github.com/Cyrix126/inquire-time

Everything seems to work in practice, even if I still need to convert the tests from chrono to time.

I was wondering, if I rework this fork to make new features "time" and "chrono", like you can choose different backends for terminal (termion, console, crossterm), if you would accept a pull request to add this features. chrono would still be the default.

My objective was to not add a new dep for my projects, as I am using time already.

**Comment by mikaelmello** [comment]
makes sense, I'd happily accept the PR!
